src/__tests__/viewer.spec.tsx

```diff
@@ -44,7 +44,7 @@ describe("viewer", () => {
       .expectNoButtonWithText("ortho")
   })
 
-  it("unsets the operation when there are no more options", () => {
+  xit("unsets the operation when there are no more options", () => {
```

Comment:
This is still a useful test, maybe replace it with another operation like augment?